<launch>

  <node pkg="LOMAS_ROS_pkg" type="watering_node.py" name="watering" output="screen">
    <parma name="watering_interval" value="120">
    <parma name="watering_time" value="20">
  </node>

  <node pkg="LOMAS_ROS_pkg" type="machine_node.py" name="machine" output="screen">
    <param name="sim_port" value="true"/>
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="path" value="/media/gcode/"/>
    <parma name="cultivation_interval" value="120">
  </node>

</launch>

